src/main/java/edu/greenblitz/robotName/subsystems/Limelight/ObjectDetectionLimelight.java

```diff
@@ -0,0 +1,37 @@
+package edu.greenblitz.robotName.subsystems.Limelight;
+
+import edu.greenblitz.robotName.VisionConstants;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+
+public class ObjectDetectionLimelight {
```

Comment:
one limelight can be used for both OD and apriltag detection by changing the `pipeline` NT value. We should by supporting that as well